pkg/products/fuse/reconciler.go

```diff
@@ -483,7 +483,7 @@ func preUpgradeBackupExecutor(rhmi *integreatlyv1alpha1.RHMI) backup.BackupExecu
 }
 
 func (r *Reconciler) reconcileSubscription(ctx context.Context, serverClient k8sclient.Client, inst *integreatlyv1alpha1.RHMI, productNamespace string, operatorNamespace string) (integreatlyv1alpha1.StatusPhase, error) {
```

Comment:
@briangallagher  Notice changes in this method that also apply to other products reconcilers.

Notice how the productNamespaceObj is not needed anymore so we replaced it with a _.

I'm/was hesitant to completely remove the code that performs resources.GetNS because I'm not sure whether this was being done due to owner being sent or because we actually want to check that that namespace exists. Being conservative I haven't removed the GetNS call.

What do you think?

Reply:
@miguelsorianod I think its fine to remove resources.GetNS. If we need to verify the namespace we can add an explicit function to do so. Everything else looks good

Reply:
Done. I've removed the calls to resources.GetNS in the reconcileSubscription methods of the products reconcilers.
I've not moved the call anywhere else because I've seen in the Reconcile method in the products there are calls to r.ReconcileNamespace so I understand the check of the namespace is implicitly done there.